src/modules/Software/List/components/SoftwareList.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+//import styles from './OsImageListItem.module.css';
+import PropsType from 'prop-types';
+//import dayjs from 'dayjs';
+import { Redirect } from 'react-router';
+import ModalDelete from '../../../../utils/components/ModalDelete';
+import api from '../../../../services/api';
+//import OsImageDropdown from '../OsImageDropdown/OsImageDropdown';
+//import SoftwareDropDown from '../actions/SoftwareDropDown';
+
+import '../styles/Software.css';
```

Comment:
Nesse projeto, foi acordado utilizar _CSS Modules_ para estilizar os componentes. Recomendo utilizar essa solução, especialmente porque você já utilizou em outros arquivos deste _pull request_.

Na verdade, eu recomendaria mesmo uma solução _CSS-in-JS_, como _styled componentes_. Mas o _CSS Modules_ é mais fácil de entender.